project_template/tasks.py

```diff
@@ -44,12 +44,58 @@ def save_verified_data(self, verified_data):
             observations=""
         )
 
+class TaskOwnedAutomobileRowEntry(DigitalizationTask):
+    task_form = forms.TranscribeOwnedAutomobileSingleRowEntry
+    template_name = "tasks/owned_automobile.html"
+    
+    def save_verified_data(self, verified_data):
+        owned_automobile, created = models.OwnedAutomobileTableEntry.object.get_or_create(
+            car_type = verified_data['tip'],
+            brand = verified_data['marca'],
+            no_owned = verified_data['numar_bucati'],
+            fabrication_year = verified_data['an_fabricatie'],
+            attainment_type = verified_data['mod_dobandire'],
+        )
 
+class TaskGetDebtsTableRowsCountEntry(DigitalizationTask):
+    task_form = forms.TranscribeDebtsTableRowsCountEntry
+    template_name = "task/owned_get_debts.html"
+        
+    def save_verified_data(self, verified_data):
+        if verified_data['nume_creditor'] and verified_data['prenume_creditor']:
+            lender_identity = "Nume: {}, Prenume: {}".format(verified_data['nume_creditor'], verified_data['prenume_creditor'])
+        else: 
+            lender_identity = "Institutie: {}".format(verified_data['institutie'])
+            
+        owned_debts, created = models.OwnedDebtsTableEntry.object.get_or_create(
+                lender=lender_identity,
+                debt_type = verified_data['tip_imprumut'],
+                acquirement_year = verified_data['an_contractare'],
+                due_value = verified_data['scadenta'],
+                value = verified_data['valoare'],
+                currency = verified_data['moneda']
+                )
+
+class TaskOwnedIncomeFromPensionsTableEntry(DigitalizationTask):
+    task_form = forms.TranscribeOwnedIncomeFromPensionsTableEntry
+    template_name = "task/owned_income_from_pensions.html"
```

Comment:
I think you may have forgotten to define these templates

https://github.com/code4romania/catpol-declaratii/pull/105/files#diff-7d71e7dc71c752440f049619dbceb02a